janusgraph-core/src/main/java/org/janusgraph/diskstorage/indexing/IndexFeatures.java

```diff
@@ -43,7 +43,7 @@ public IndexFeatures(boolean supportsDocumentTTL, Mapping defaultMap, ImmutableS
                          String wildcardField, ImmutableSet<Cardinality> supportedCardinaities, boolean supportsNanoseconds,
                          boolean supportCustomAnalyzer, boolean supportsGeoContains) {
 
-        Preconditions.checkArgument(defaultMap!=null || defaultMap!=Mapping.DEFAULT);
```

Comment:
Thanks. I guess `defaultMap==Mapping.DEFAULT` wouldn't make sense. Incidentally I don't see any usages of the defaultMap and (accessor is not covered according to [coverity](https://codecov.io/gh/JanusGraph/janusgraph/src/7608d3325fe5d531647c6c9677fca54701ecdb93/janusgraph-core/src/main/java/org/janusgraph/diskstorage/indexing/IndexFeatures.java#L63...65)).